grype/presenter/cyclonedx/vulnerability.go

```diff
@@ -91,24 +103,17 @@ func NewVulnerability(m match.Match, p vulnerability.MetadataProvider) (Vulnerab
 		}
 		score.Base = cvss.Metrics.BaseScore
 		rating.Score = score
+		rating.Severity = severity
+
 		method, err := cvssVersionToMethod(cvss.Version)
-		if err != nil {
+		if err == nil {
+			rating.Method = method
+			rating.Vector = cvss.Vector
+		} else {
 			log.Errorf("unable to parse CVSS version: %v", err)
 			// do not halt execution if one CVSS fails to provide an accurate Version
-			continue
```

Comment:
was dropping this continue statement intentional?

Reply:
Yes, so the rating can still be appended with the severity and score (on the assumption that partial data is better than no data)